### DISCUSSION TITLE
Autostop Listenin

### DISCUSSION THREAD

**Discussion by kokucoding** [AndraxDev/speak-gpt#130] [Ideas]
Maybe you should add an auto-stop listening feature if the user is quiet for a certain time.

**Comment by AndraxDev** [comment]
With whisper API it is not possible to do. There's no technical implementation to do so.

**Reply by thiswillbeyourgithub** [reply]
FYI there's a minimalist whisper keyboard project that implemented it I think : https://github.com/j3soon/whisper-to-input

**Comment by AndraxDev** [comment]
Closing and locking this discussion due to inactivity.
